pyspark/test/bigdl/test_simple_integration.py

```diff
@@ -320,7 +320,7 @@ def gen_rand_sample():
 
         assert parameters["linear1"] is not None
         print("parameters %s" % parameters["linear1"])
-        predict_result = trained_model.predict(trainingData)
+        predict_result = trained_model.predict(trainingData, batch_size = 4)
```

Comment:
should we need to add a new test case instead of replace the original test batch_size = -1 case